src/Controls/src/Core/Platform/iOS/Extensions/TextExtensions.cs

```diff
@@ -72,8 +72,14 @@ public static void UpdateMaxLines(this UILabel platformLabel, Label label)
 		internal static void SetLineBreakMode(this UILabel platformLabel, Label label)
 		{
 			int maxLines = label.MaxLines;
+
 			if (maxLines < 0)
-				maxLines = 0;
+			{
+				if (label.LineBreakMode == LineBreakMode.TailTruncation)
+					maxLines = 1;
+				else
+					maxLines = 0;
+			}
```

Comment:
Do we need to do this? We already set to 1 in the switch.

Reply:
Moved to the switch.